package.json

```diff
@@ -13,7 +13,7 @@
   "engines": {
     "node": ">=8.9.0",
     "npm": ">=5.5.1",
-    "yarn": ">=1.3.2"
+    "yarn": ">=1.5.0"
```

Comment:
yarn 1.3.2 wasn't hoisting transitive dependencies to the top level,which caused unit tests to fail